sklearn/tests/test_docstring_parameters.py

```diff
@@ -143,7 +148,9 @@ def test_tabs():
                         'feature_extraction._hashing_fast' in modname):
             continue
 
-        # because we don't import
+        # Do not check deprecated paths
+        if modname in DEPRECATED_MODULE_NAMES:
```

Comment:
why do we need this? There shouldn't be any tab anyway?

Reply:
The `Pep562` wraps the original module and is an instance, which means `getsource` will fail in this case.

I updated this PR by unwrapping the instance and grab the underlying `_module`.